pkg/repository/notificationrepository/notification_db_models.go

```diff
@@ -6,15 +6,15 @@ package notificationrepository
 
 import (
 	"encoding/json"
-
 	"github.com/greenbone/opensight-notification-service/pkg/helper"
 	"github.com/greenbone/opensight-notification-service/pkg/models"
+	"github.com/greenbone/opensight-notification-service/pkg/services/notificationservice/dtos"
 )
 
 const notificationsTable = "notification_service.notifications"
 
 const createNotificationQuery = `INSERT INTO ` + notificationsTable + ` (origin, origin_uri, timestamp, title, detail, level, custom_fields) VALUES (:origin, :origin_uri, :timestamp, :title, :detail, :level, :custom_fields) RETURNING *`
-const unfilteredListNotificationsQuery = `SELECT * FROM ` + notificationsTable
+const unfilteredListNotificationsQuery = `SELECT * FROM ` + notificationsTable + ` `
```

Comment:
the trailing space should not be necessary here, it should be instead added wherever something is appended to this query

Reply:
Removed now.